3600/3/app/views/home.erb

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
```

Comment:
Layouts are using for avoiding a lot of code within view files

Reply:
ERB is a preprocessor, so the file should be named like `my_awesom_view.html.erb`